llama_stack/providers/remote/inference/nvidia/config.py

```diff
@@ -47,10 +47,15 @@ class NVIDIAConfig(BaseModel):
         default=60,
         description="Timeout for the HTTP requests",
     )
+    append_api_version: bool = Field(
+        default=lambda: os.getenv("NVIDIA_APPEND_API_VERSION", True),
```

Comment:
is the only way to disable api appending to set `NVIDIA_APPEND_API_VERSION=""`?

if so, the result of getenv should be coerced to a bool.